test/unit/test_vector.c

```diff
@@ -723,51 +750,51 @@ static bool test_pvector_insert() {
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 1)), (void *)1337, "insert => content");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 2)), (void *)1, "insert => old content");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 3)), (void *)2, "insert => old content");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 4)), (void *)3, "insert => old content");
 	r_pvector_clear (&v);
 
 	init_test_pvector2 (&v, 4, 0);
 	e = (void *)1337;
 	e = *r_pvector_insert (&v, 1, e);
 	mu_assert ("insert (resize) => capacity", v.v.capacity >= 5);
-	mu_assert_eq_fmt (v.v.len, 5UL, "insert (resize) => len", "%lu");
-	mu_assert_eq_fmt (e, (void *)1337, "insert (resize) => content at returned ptr", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 0)), (void *)0, "insert (resize) => old content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 1)), (void *)1337, "insert => content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 2)), (void *)1, "insert => old content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 3)), (void *)2, "insert => old content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 4)), (void *)3, "insert => old content", "%p");
+	mu_assert_eq (v.v.len, 5UL, "insert (resize) => len");
+	mu_assert_eq (e, (void *)1337, "insert (resize) => content at returned ptr");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 0)), (void *)0, "insert (resize) => old content");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 1)), (void *)1337, "insert => content");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 2)), (void *)1, "insert => old content");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 3)), (void *)2, "insert => old content");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 4)), (void *)3, "insert => old content");
 	r_pvector_clear (&v);
 
 	init_test_pvector2 (&v, 4, 2);
 	e = (void *)1337;
 	e = *r_pvector_insert (&v, 4, e);
-	mu_assert_eq_fmt (v.v.len, 5UL, "insert (end) => len", "%lu");
-	mu_assert_eq_fmt (e, (void *)1337, "insert (end) => content at returned ptr", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 0)), (void *)0, "insert (end) => old content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 1)), (void *)1, "insert (end) => old content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 2)), (void *)2, "insert (end) => old content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 3)), (void *)3, "insert (end) => old content", "%p");
-	mu_assert_eq_fmt (*((void **)r_vector_index_ptr (&v.v, 4)), (void *)1337, "insert (end) => content", "%p");
+	mu_assert_eq (v.v.len, 5UL, "insert (end) => len");
+	mu_assert_eq (e, (void *)1337, "insert (end) => content at returned ptr");
+	mu_assert_eq (*((void **)r_vector_index_ptr (&v.v, 0)), (void *)0, "insert (end) => old content");
```

Comment:
just FYI (you can keep it this way here) there is also `mu_assert_ptreq` and `mu_assert_ptrneq`.